app/Schema/user_schema.py

```diff
@@ -1,31 +1,30 @@
 from app import ma
 from marshmallow.validate import Length, Regexp
-from marshmallow import fields, INCLUDE
+from marshmallow import fields
 from app.model.user import User
 
-
 name_string = '^[a-zA-Z]*$'
+address_string = '^[a-zA-Z ]*$'
 
 
 class UserSchema(ma.Schema):
     """Adding the Schema validations to the User """
 
     first_name = fields.Str(required=True, validate=(Length(max=50), Regexp(name_string)))
     last_name = fields.Str(required=True, validate=(Length(max=50), Regexp(name_string)))
-    address = fields.Str(required=True, validate=Length(max=100))
-    mobile_number = fields.String(required=True, validate=Regexp('[7-9][0-9]{9}'))
+    address = fields.Str(required=True, validate=(Length(max=100), Regexp(address_string)))
+    mobile_number = fields.String(required=True, validate=(Regexp('^[789]\d{9}$'), Length(equal=10)))
     email_id = fields.Email(required=True, validate=Regexp('^[a-z0-9]+[\._]?[a-z0-9]+[@]\w+[.]\w{2,3}$'))
```

Comment:
With `fields.Email` do we need to add separate `regexp` additionally?

Reply:
No, its not required have removed the Regex pattern now .